InterfaceMATLAB/tsgCoreTests.m

```diff
@@ -997,6 +997,62 @@ function tsgCoreTests()
 
 disp(['Utility functions:        PASS']);
 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%                     tsgDifferentiate()                           %%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+[lGrid, p] = tsgMakeGlobal('_tsgcoretests_diff', 2, 3, 'leja', 'iptotal', 3);
+num_p = length(p);
+f = @(x) [x(1), x(1)*x(2)*x(2), x(2)*x(2)];
+Df = @(x) [1, 0; x(2)*x(2), 2*x(1)*x(2); 0, 2*x(2)];
```

Comment:
Can you check if this works on Octave. I suspect it does, but check anyway.

Reply:
What did you change specifically. If you must, don't use function handles but copy-paste the formulas. It may be a bad approach, but I don't want to drop support for either platform just because of two lines of code in testing.

Reply:
The function handles were fine. The specific changes were

- `norm(x, 'F')` to `norm(x, 'fro')`
- `rng(seed)` to `rand('twister', seed)`

Only the latter is deprecated and not recommended by MATLAB.